lib/util/spmconfig.js

```diff
@@ -46,14 +61,26 @@ var envFileName = '/etc/sematext/receivers.config'
 **/
 loadEnvFromFile(envFileName)
 
+if (process.env.REGION !== undefined) {
+  setReceiverUrlsForRegion(process.env.REGION)
+}
+
 var spmDefaultConfig = {
   tokens: {
     spm: process.env.SPM_TOKEN || process.env.MONITORING_TOKEN,
     // prepare move to MONITORING_TOKEN and config.token.monitoring
     monitoring: process.env.MONITORING_TOKEN || process.env.SPM_TOKEN
   },
   influx: {
-    dbName: 'metrics'
+    dbName: 'metrics',
```

Comment:
Do you have a dbName for tag aliases (name is `tagAliases`)?

Reply:
The tag alias interface is not implemented yet.  Would it need a different dbName? Anyhow, i assume we will use not the official influx client for tagAlias feature.

Reply:
Yes :), the name is - `tagAliases`. Mentioning since you asked earlier today about the format.